Makefile

```
@@ -53,3 +56,8 @@ tag: update-kfdef-with-tag
 	git tag $(TAG_NAME)
 push-tag:
 	git push --tags
+
+# requires kube-linter to be installed (https://github.com/stackrox/kube-linter)
+lint:
```

Can you change this target to `kube-lint`? 

```suggestion
kube-lint:
```